ansible/roles/test/files/ptftests/sad_path.py

```diff
@@ -20,12 +20,13 @@ def __init__(self, oper_type, vm_list, portchannel_ports, vm_dut_map, test_args,
         self.shandle = SadOper(self.oper_type, self.vm_list, self.portchannel_ports, self.vm_dut_map, self.test_args, self.dut_ssh)
 
     def setup(self):
-        if 'bgp' in self.oper_type:
-            self.shandle.sad_setup(is_up=False)
+        self.shandle.sad_setup(is_up=False)
         return self.shandle.retreive_test_info(), self.shandle.retreive_logs()
 
-    def verify(self):
+    def verify(self, pre_check=True):
```

Comment:
>pre_check [](start = 21, length = 9)

The variable is effective conditionally, suggest name it as ''sad_lag_pre_check"

Reply:
This var is not specific to the LAG test. I will be using it for other preboot types as well. I am using this var to tag the failed msg with Preboot/Postboot since the failed msgs are all consolidated and printed at the end of the test. Hard to figure out when the failure occurred since the same method is used for both preboot and postboot checks.